.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: continuous-integration
+
+on: [pull_request , push]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest tables
+        pip install pytest-cov
+        python setup.py develop
+        pip install -U pip nose coverage numpy scipy pandas numba sympy ipython statsmodels flake8 
+        python setup.py install
+        pip install coveralls
+    - name: Run Tests
+      
+      env:
+          COVERALLS_REPO_TOKEN: 7f8JDNk6BsPMESy0URL3W00zlazzRcuzR 
```

Comment:
Can you remind me what this REPO token is for?

Is this for `travis` to post the `coverage` results back to the REPO (or something else)?

We may need to rework this part for `GitHub Actions`.